docs/src/main/sphinx/connector/hive.md

```diff
@@ -240,6 +240,15 @@ Hive connector documentation.
     `query_partition_filter_required` catalog session property for temporary,
     catalog specific use.
   - `false`
+* - `hive.query-partition-filter-required-schemas`
```

Comment:
Please follow the commit message guideline: https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#format-git-commit-messages

Reply:
Sure, I squashed the commits with a message representing this PR.

Reply:
Reminder. Please capitalize and shorten the commit title.